app/controllers/security_group_controller.rb

```diff
@@ -198,6 +209,10 @@ def update_finished
     end
   end
 
+  def display_instances
+    nested_list(VmOrTemplate, :association => :vms, :breadcrumb_title => _("Instances"))
```

Comment:
`VmCloud` instead of `VmOrTemplate` should fix the 4 "SecurityGroupController for * #show show associated instances" spec failures.

That's because `VmOrTemplate` means infra VMs & templates, cloud instances and images, while VmCloud is specifically instances.

Reply:
Although, instances was already in the list, so the original `display_instances` implementation from `GenericShowMixin` should work.

Any reason to replace it? (I think you can just delete this copy of the method.)